gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -54,8 +54,12 @@ class {{ service.async_client_name }}:
 
     _client: {{ service.client_name }}
 
+    # Copy defaults from the synchronous client for use here.
+    # Note: DEFAULT_ENDPOINT is deprecated. Use DEFAULT_ENDPOINT_TEMPLATE instead.
     DEFAULT_ENDPOINT = {{ service.client_name }}.DEFAULT_ENDPOINT
     DEFAULT_MTLS_ENDPOINT = {{ service.client_name }}.DEFAULT_MTLS_ENDPOINT
+    DEFAULT_ENDPOINT_TEMPLATE = {{ service.client_name }}.DEFAULT_ENDPOINT_TEMPLATE
```

Comment:
Does this symbol really need to be public? If not, let's underscore it  (here and in the sync_client)